### ISSUE TITLE
[1.19.2] Server Crash onCrafted

### ISSUE THREAD

**Issue by itsdinkd** [blackd/Inventory-Profiles#283]
This is a dedicated server crash for the official Another Quality Modpack 2 server.

https://gist.github.com/itsdinkd/3783e71c1670ec12703ca19041974fc2

Mapped stacktrace
```java
java.lang.Error: Watchdog
	at java.base@17.0.6/java.lang.invoke.LambdaForm$MH/0x00000008051cc000.invoke(LambdaForm$MH)
	at java.base@17.0.6/java.lang.invoke.LambdaForm$MH/0x0000000802db6000.invokeExact_MT(LambdaForm$MH)
	at java.base@17.0.6/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:732)
	at dev.architectury.event.EventFactory.invokeMethod(EventFactory.java:53)
	at dev.architectury.event.EventFactory$1.handleInvocation(EventFactory.java:62)
	at com.google.common.reflect.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:89)
	at jdk.proxy2/jdk.proxy2.$Proxy194.craft(Unknown Source)
	at net.minecraft.screen.slot.CraftingResultSlot.handler$bak000$craft(CraftingResultSlot.java:1548)
	at net.minecraft.screen.slot.CraftingResultSlot.onCrafted(CraftingResultSlot.java:47)
	at net.minecraft.screen.slot.CraftingResultSlot.onCrafted(CraftingResultSlot.java:36)
	at net.minecraft.screen.slot.Slot.onQuickTransfer(Slot.java:30)
	at net.minecraft.screen.PlayerScreenHandler.transferSlot(PlayerScreenHandler.java:180)
	at net.minecraft.screen.ScreenHandler.internalOnSlotClick(ScreenHandler.java:396)
	at net.minecraft.screen.ScreenHandler.onSlotClick(ScreenHandler.java:293)
	at net.minecraft.server.network.ServerPlayNetworkHandler.onClickSlot(ServerPlayNetworkHandler.java:1770)
	at net.minecraft.network.packet.c2s.play.ClickSlotC2SPacket.apply(ClickSlotC2SPacket.java:58)
	at net.minecraft.network.packet.c2s.play.ClickSlotC2SPacket.apply(ClickSlotC2SPacket.java:13)
	at net.minecraft.network.NetworkThreadUtils.method_11072(NetworkThreadUtils.java:22)
	at net.minecraft.class_2600$$Lambda$31998/0x0000000804f2d160.run(Unknown Source)
	at net.minecraft.server.ServerTask.run(ServerTask.java:18)
	at net.minecraft.util.thread.ThreadExecutor.executeTask(ThreadExecutor.java:157)
	at net.minecraft.util.thread.ReentrantThreadExecutor.executeTask(ReentrantThreadExecutor.java:23)
	at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:776)
	at net.minecraft.server.MinecraftServer.executeTask(MinecraftServer.java:159)
	at net.minecraft.util.thread.ThreadExecutor.runTask(ThreadExecutor.java:131)
	at net.minecraft.server.MinecraftServer.runOneTask(MinecraftServer.java:758)
	at net.minecraft.server.MinecraftServer.runTask(MinecraftServer.java:752)
	at net.minecraft.util.thread.ThreadExecutor.runTasks(ThreadExecutor.java:116)
	at net.minecraft.server.MinecraftServer.runTasksTillTickEnd(MinecraftServer.java:736)
	at net.minecraft.server.MinecraftServer.handler$ckg000$modifiedRunLoop(MinecraftServer.java:7894)
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:644)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257)
	at net.minecraft.server.MinecraftServer$$Lambda$28915/0x00000008047d4f00.run(Unknown Source)
	at java.base@17.0.6/java.lang.Thread.run(Thread.java:833)
```

**Comment by blackd** [comment]
I fail to see where in the logs is Inventory Profiles Next is even mentioned.

Also IPN is client side mod so if you even have it installed on the server it does nothing.

**Comment by itsdinkd** [comment]
I'm not a Java developer so I'm pretty amatuer when reading stack traces
however I cross referenced classes in the stack and IPN popped up.
OnCrafted and CraftingSlot whatever (I'm on my phone and replying to email)
but it's what's in 2 of the first 5 lines.

**Comment by itsdinkd** [comment]
CraftingResultSlot and onCrafted, I think it was also onQuickTransfer. I looked these up on github and the only mod in aqm2 that uses these is IPN. 

Edit: it was onCrafted

**Comment by blackd** [comment]
you have attached wrong logs then. There is nothing IPN related in them.

But if you have a server and you are sure that IPN causes the problem just remove it.

**Comment by blackd** [comment]
I mean if the server crashes and you have IPN in the server mods folder. You can just remove IPN and libIPN from said mods folder.

**Comment by blackd** [comment]
fixed in 1.9.3/4
